deployment/docker-compose.yml

```diff
@@ -1,5 +1,24 @@
 version: "3.7"
 services:
+  aerie_commanding:
+    container_name: aerie_commanding
+    depends_on: ["postgres"]
+    environment:
+      LOG_FILE: console
+      LOG_LEVEL: warn
+      MERLIN_GRAPHQL_URL: http://hasura:8080/v1/graphql
+      COMMANDING_SERVER_PORT: 3000
```

Comment:
In a near-future commit we should change the port to match more closely with the other servers ports

Reply:
Is there anything that would prohibit us doing that as part of this PR?

Reply:
Change made.